replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+
+```{r echo = FALSE}
+# high-income & small account : expensiveness
+df4_spss %>%
+  filter(split == 1, largeacc == 0) %>%
+  summarise(mean.expense.highincome.smallaccount = mean(expense), CI.low = mean(expense) - (1.96 * Sem(expense)), CI.high = mean(expense) + (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+
+# high-income & large account : expensiveness
+df4_spss %>%
+  filter(split == 1, largeacc == 1) %>%
+  summarise(mean.expense.highincome.largeaccount = mean(expense), CI.low = mean(expense) - (1.96 * Sem(expense)), CI.high = mean(expense) + (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+
+```
+
+> whereas the evaluations of lower-income participants did not differ significantly between the conditions (small account: M = 5.55, 95% CI = [4.92, 6.18]; large account: M = 5.75, 95% CI = [5.03, 6.48]).
+
+```{r echo = FALSE}
+# low-income & small account : expensiveness
+df4_spss %>%
+  filter(split == 0, largeacc == 0) %>%
+  summarise(mean.expense.lowincome.smallaccount = mean(expense), CI.low = mean(expense) - (1.96 * Sem(expense)), CI.high = mean(expense) + (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+
+# low-income & large account : expensiveness
+df4_spss %>%
+  filter(split == 0, largeacc == 1) %>%
+  summarise(mean.expense.lowincome.largeaccount = mean(expense), CI.low = mean(expense) - (1.96 * Sem(expense)), CI.high = mean(expense) + (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+
+```
+
+
+
+```{r echo = FALSE}
+df4_high_small <- df4_spss %>%
+  filter(split == 1, largeacc == 0) %>%
+  summarise(mean.expense.highincome.smallaccount = mean(expense), CI = (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+
+df4_high_large <- df4_spss %>%
+  filter(split == 1, largeacc == 1) %>%
+  summarise(mean.expense.highincome.largeaccount = mean(expense), CI = (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (4) - high income
+```
+
+```{r echo = FALSE}
+df4_low_small <- df4_spss %>%
+  filter(split == 0, largeacc == 0) %>%
+  summarise(mean.expense.lowincome.smallaccount = mean(expense), CI = (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+
+df4_low_large <- df4_spss %>%
+  filter(split == 0, largeacc == 1) %>%
+  summarise(mean.expense.lowincome.largeaccount = mean(expense), CI = (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (4) - low income
+```
+
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
+values = c(df4_low_small$mean.expense.lowincome.smallaccount, df4_high_small$mean.expense.highincome.smallaccount, df4_low_large$mean.expense.lowincome.largeaccount, df4_high_large$mean.expense.highincome.largeaccount)
+account_size = c("small_account", "small_account", "large_account", "large_account")
+CIs = c(df4_low_small$CI, df4_high_small$CI, df4_low_large$CI, df4_high_large$CI)
+
```

Comment:
See previous comments about using "<-" versus an equal sign for variable assignment.